http/vertx-web-client/src/test/java/io/quarkus/qe/vertx/webclient/PureVertxHttpClientTest.java

```diff
@@ -18,8 +18,7 @@
 import io.vertx.mutiny.ext.web.client.predicate.ResponsePredicate;
 
 @QuarkusTest
-//TODO: https://github.com/quarkusio/quarkus/issues/23865
-@Disabled
+@Tag("fips-incompatible") // Reported in https://github.com/IBM/Db2/issues/43
```

Comment:
```suggestion
@Tag("fips-incompatible") // Reported in https://github.com/quarkusio/quarkus/issues/23865
```
Shouldn't there be the original issue link?